projects/vdk-plugins/vdk-csv/tests/functional/test_csv_plugin.py

```diff
@@ -76,3 +80,46 @@ def test_ingestion_csv_with_options():
 
     assert len(ingest_plugin.payloads) > 0
     assert ingest_plugin.payloads[0].payload[0]["US Zip"] == "08056"
+
+
+def test_csv_export(tmpdir):
```

Comment:
Does the test cover the full 'export_csv' method e.g do we hit the 'Already existing path: {fullpath}. Try again with another path or file name.' log call? I would try to cover as much of the method as possible (tests hit all lines)